docs/tutorial/01-getting-started.md

```diff
@@ -0,0 +1,367 @@
+        a_number == env.storage().instance().get::<_, u64>(&THE_NUMBER).unwrap()
+    }
+}
+```
+
+Finally, we add the `guess` function which accepts a number as the guess and compares it to the stored number, returning the result. Notice we're using our defined key (that small Symbol) to find stored data that may or may not be there. That's why we need [`unwrap()`](https://doc.rust-lang.org/rust-by-example/error/option_unwrap.html), but we'll talk more about `Option` values later in the tutorial.
+
+```rust
+mod test;
+```
+
+Post Script: this last line includes the test module into this file. It's handy to write unit tests for our code in a separate file (`contracts/guess-the-number/src/test.rs`), but you could also write them inline if you want.
+
+### 👷‍♀️ Let's Make a Change
+
+We should still have our original `npm start` command running. I told you it did a lot of heavy lifting for you, but it also updates all of that automatically whenever you make changes to your code. Let's test it out by making a small change and watch the dev server update immediately.
+
+The docstring for our `guess` function says to guess a number "between 1 and 10". But does that include "10"? Let's clarify:
+
+```rust
+  /// Guess a number between 1 and 10, inclusive
+  pub fn guess(env: &Env, a_number: u64) -> bool {
+```
+
+Save the file and watch your terminal output. The contracts get rebuilt, redeployed, and clients for them get regenerated for your frontend. Then Vite hot-reloads your app and you should see the change in the contract explorer in your browser.
+
+Tada!
+
+### 👀 Looking Deeper at the Output
+
+Sharp eyed readers might have noticed interesting lines in the build output. Let's take a look:
+
+```
+[0] ℹ️ Starting local network
+```
+
+This is the Docker container running Stellar locally. You can manage it with `stellar container` commands, but Scaffold Stellar should handle everything for you. We recommend using the local network for development because it's simpler than working with [the other networks](https://developers.stellar.org/docs/learn/fundamentals/networks), but we'll make use of them in later parts of the tutorial.
+
+```
+ℹ️ Build Summary:
+[0]   Wasm File: target/stellar/local/guess_the_number.wasm
+[0]   Wasm Hash: 2cda774ed515acb3af208478e64841d6253d52366f22156c5e7a550ee7b06cbe
+[0]   Exported Functions: 5 found
+[0]     • _
+[0]     • __constructor
+[0]     • guess
+[0]     • reset
+[0]     • upgrade
+[0] ✅ Build Complete
+```
+
+You'll see a build summary for each of our contracts as they compile to Wasm, the code that will actually run on the blockchain. There's a few other functions in our contract that weren't exported here. That's because they're private. We'll implement our own in the next part of the tutorial and explain why they're useful.
+
+```
+[0] ℹ️ Creating keys for "me"
+[0] ✅ Key saved with alias me in "/Users/zach/code/guessing-game-tutorial/.config/stellar/identity/me.toml"
```

Comment:
I updated so that we use the global config to ensure that `me` can be found outside the project and other identities can also be found in the project.